apps/prairielearn/src/lib/db-types.ts

```diff
@@ -101,3 +101,19 @@ export const UserSchema = z.object({
   user_id: IdSchema,
 });
 export type User = z.infer<typeof UserSchema>;
+
+export const PlanGrantSchema = z.object({
+  course_instance_id: IdSchema.nullable(),
+  created_at: z.date(),
+  // TODO: handled `deleted_at` everywhere we need to.
```

Comment:
👀 - do we actually want soft deletes here?

Related: should we be recording audit logs plan grants, enrollment limits, etc?

Reply:
Soft deletes removed in 3d49ac2.